.github/workflows/main.yml

```diff
@@ -16,11 +16,11 @@ jobs:
       matrix:
         include:
           - pair:
-              elixir: 1.13.4
-              otp: 24.3
+              elixir: "1.13"
+              otp: "24"
           - pair:
-              elixir: 1.17.2
-              otp: 27.0
+              elixir: "1.18"
+              otp: "27"
```

Comment:
Specify versions as string, see https://github.com/erlef/setup-beam?tab=readme-ov-file#specify-versions-as-strings-not-numbers